gitlab.json

```diff
@@ -4,8 +4,8 @@
   "artifact": "https://github.com/freenas/iocage-plugin-gitlab.git",
   "pkgs": [
     "www/gitlab-ce",
-    "databases/postgresql95-server",
-    "databases/postgresql95-contrib",
+    "databases/postgresql11-server",
```

Comment:
Do we have this package in 11.2 packagesite ? It seems we don't ?

Reply:
Or is `postgresql11-client-11.5.txz` sufficient ?

Reply:
Gitlab is currently removed from the index because is broken, I will push these packages once all confirmed working